keras_nlp/models/roberta/roberta_classifier.py

```diff
@@ -194,6 +194,14 @@ def __init__(
         self.hidden_dim = hidden_dim
         self.dropout = dropout
 
+        # Default compilation
+        self.compile(
+            loss=keras.losses.SparseCategoricalCrossentropy(from_logits=True),
+            optimizer=keras.optimizers.Adam(2e-5),
```

Comment:
How did we come to the lower number for RoBERTa than BERT?

Reply:
This was from Chen's GLUE testing ([link](https://github.com/keras-team/keras-nlp/blob/master/examples/glue_benchmark/scores.md#hyperparameter-settings)). However @chenmoneygithub has said that `Adam(5e-5)` seems to work well for most encoder-only models, so happy to apply that everywhere.

Reply:
yea 2e-5 works a little better in Roberta in my GLUE experiment.